pysumma/plotting/layers.py

```diff
@@ -7,8 +7,9 @@
 from .utils import justify
 
 
-def layers(var, depth, ax=None, colormap='viridis',
-           plot_soil=False, plot_snow=True, variable_range=None, **kwargs):
+def layers(var, depth, ax=None, colormap='viridis', plot_soil=False,
+           plot_snow=True, variable_range=None, add_colorbar=True,
```

Comment:
I think `add_colorbar` must be False as default. So if we set `add_colorbar` in `layers` method and we need to set `car_kwargs` together, we can apply this setting in line 63.

Reply:
I'm not quite sure what you mean here. I think it makes more sense to have the default to draw a color bar, since otherwise it's impossible to know the values of what's being plotted. This also somewhat mirrors the plotting abilities to things like `xarray`'s `DataArray.plot.pcolormesh` functionality. This design has the added benefit of the user not having to remember to set `add_colorbar` every time they set some custom color bar configuration through `cbar_args`.